src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -224,20 +254,19 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   createLogger(configuration: MeetingSessionConfiguration) {
-    const { logLevel, postLogConfig } = this.config;
-    const consoleLogger = new ConsoleLogger('SDK', logLevel);
+    const consoleLogger = new ConsoleLogger('SDK', this.logLevel);
```

Comment:
can we also use this.meetingManagerConfig here instead of this.logLevel and this.postLoggerConfig?  Also maybe we want to add a comment to say these variables are kept for compatibility reason and will be removed in next major version update.

Reply:
Added comment to variables as well as in the method, this is a good call, thanks!